python/lsst/ts/mtdome/mock_llc/mock_motion/shutter_motion.py

```diff
@@ -72,43 +72,15 @@ def __init__(
             start_tai=start_tai,
         )
         self.log = logging.getLogger("ShutterMotion")
-        # Keep track of being in error state or not.
-        self.motion_state_in_error = False
         # Keep track of which drives are in error state.
         self.drives_in_error_state = [False] * NUM_MOTORS_PER_SHUTTER
-        # Keep track of when motion state went into ERROR
-        self._error_start_tai = 0.0
-        # Keep track of the position at the moment that ERROR state starts.
-        self._error_state_position = 0.0
 
     def _get_duration(self) -> float:
```

Comment:
Is this method still needed? I hope you can delete it (short though it is).

Reply:
Yes it is. This class doesn't use angles (in radians) but a percentage. It cannot use the superclass method. I have added a comment explaining that.